File: Java Programming/Pratice of Java Programming/program3.java
/*
3. Accept number from user and display below pattern.
Input : 5
Output : 1 * 2 * 3 * 4 * 5
*/

import java.util.Scanner;

class Logic
{
    void Pattern(int iNo)
    {
        int i =0;

        for(i=1; i<=iNo; i++)
        {
            System.out.print(""+i);
             if (i<iNo)
                {
                 System.out.print("\t*\t");
                }
        }
    }
}

class program3 
{
    public static void main(String[] args) 
    {
        Scanner sobj=new Scanner(System.in);

        int iValue =0;

        System.out.print("Enter number of elements: ");
        iValue=sobj.nextInt();

        Logic obj=new Logic();

        obj.Pattern(iValue);

    }
    
}

File: C Programming/Competitive Programming/Coding Challenges 19/program3.c
/*
Q3. Accept number of rows and number of columns from user and display below
pattern.

Input : iRow = 5 iCol = 5
Output :$ * * * *
        * $ * * *
        * * $ * *
        * * * $ *
        * * * * $
*/
#include<stdio.h>

void Pattern(int iRow,int iCol)
{
    int i=0;
    int j=0;

    for ( i = 1; i <= iRow; i++)
    {
        for ( j = 1; j <= iCol; j++)
        {
            if (j==i)
            {
                printf("$ ");
            }
            else
            {
                printf("* ");
            }
            
        }
        printf("\n");
    }
    
}

int main()
{
    int iValue1=0,iValue2=0;

    printf("Enter The Number of Row: ");
    scanf("%d",&iValue1);

    printf("Enter The Number of Columns: ");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}